lowell-dashboard/app/templates/profile.py

```diff
@@ -26,9 +26,9 @@
   </nav>
   
   <section id="bio">    
-    <p>{{user.first_name}}</p>
-    <p>{{user.last_name}}</p>
-    <p>{{user.email}}</p>
+    <p>First name: {{user.first_name}}</p>
+    <p>Last name: {{user.last_name}}</p>
+    <p>Email: {{user.email}}</p>
```

Comment:
We probably shouldn't show that unless the user allows this in their profile page view with their options.

Reply:
Yep. Remove the first and last names and replace it with something else.